gensim/models/word2vec.py

```diff
@@ -1600,7 +1602,7 @@ def accuracy(self, questions, restrict_vocab=30000, most_similar=most_similar, u
                 if not section:
                     raise ValueError("missing section header before line #%i in %s" % (line_no, questions))
                 try:
-                    if use_lowercase:
+                    if case_insensitive:
```

Comment:
Yes, I think the "skipping invalid line" log-message should be followed with a continue to definitely skip.